### ISSUE TITLE
Website: Login broken

### ISSUE THREAD

**Issue by markus2330** [ElektraInitiative/libelektra#1679]
@Namoshek wrote:
> The login does work, the token is stored and the user is authenticated (which is also visible after refreshing the page). But for some reason, both the `.then(function(response) { ... })` and the `.catch(function(response) { ... })` functions are called in the [AuthLoginController.js](../blob/master/src/tools/rest-frontend/resources/assets/js/controllers/main/auth/AuthLoginController.js). The login method on the authentication service is called as described in [their API](https://github.com/sahat/satellizer#authloginuser-options) though. And there have been no changes to the satellizer project since mid 2016, which was before we had a working build. So I'm pretty much clueless.

What about disabling login? Might be better than to have a non-working login?

**Comment by markus2330** [comment]
I am considering to mention the snippet sharing (and the more general topic that (mis)configurations should be treated as free software and should be shared)  but it would not be wise to recommend something where the login does not work.

Any plans to reactivate the login? Or allow anonymous posting of snippets? Or use the new version from @vanessaNWK ?

**Comment by Namoshek** [comment]
It looks like one of the dependencies is causing the (display-only) issue with the login. The `angular-ui-notification` package was changed due to the same requirements of the `angular` core package as have been done in #1678. What I don't understand is why still the old version of the package is used in the build, because we used `"angular-ui-notification": "^0.2.0"` in the dependency tree, what means it should use all newer versions up to, but excluding `1.0.0`. And the newest is `0.3.6` (for quite some time already).

As my development VM is not working anymore after upgrading to Windows 10 some time ago (I guess it has to do with me enabling Hyper-V for Docker...), it is not very easy for me to test things locally. I created a PR with a minor change (#1772) that potentially fixes the issue. I hope it does not introduce other compatibility issues though.
